### ISSUE TITLE
Implement the `stop-screensaver` option

### ISSUE THREAD

**Issue by matteocoder** [mpv-android/mpv-android#567]
I noticed that even with `stop-screensaver=no` set in `mpv.conf`, mpv still prevents the screen from going to sleep during playback,
and the feature does not appear in the version information window.

Adding the option could be a nice complement to the background playback functionality for people who use mpv as an audio player.

Thank you very much for your work

### Additional information

Android 8.1, mpv-android 2022-03-23.

**Comment by sfan5** [comment]
The feature is hardcoded into the Java part since libmpv has no way to handle it, it would be possible to reimplement the option inside Java.

> Adding the option could be a nice complement to the background playback functionality for people who use mpv as an audio player.

I was under the impression the keep-screen-on flag would only apply when the mpv application is visible, is this not the case?

**Comment by matteocoder** [comment]
> I was under the impression the keep-screen-on flag would only apply when the mpv application is visible, is this not the case?
Yes, functionally it should only apply when mpv is visible. I was referring to the fact that since mpv-android can already play back files in the background, adding the ability to sleep the screen when listening to audio would be a nice quality of life improvement.

By the way, does mpv-android support conditional auto profiles like the the desktop program?

**Comment by sfan5** [comment]
Yes, auto-profiles.lua works by default
